Types.ts

```diff
@@ -291,7 +291,9 @@ export interface LinkError {
     errorCode: LinkErrorCode;
     errorType: LinkErrorType;
     errorMessage: string;
+    /** @deprecated Use displayMessage instead */
     errorDisplayMessage?: string;
```

Comment:
Remove this in the next major update https://jira.plaid.com/browse/RNSDK-109

Reply:
Just curious: where does the `@deprecated` come from? Is it just something we added, or does it have IDE support in some IDEs?

Reply:
https://jsdoc.app/tags-deprecated.html
It's a jsdoc tag